src/components/organisms/SddWidget.tsx

```diff
@@ -0,0 +1,7 @@
+const SddWidget = () => {
+  return (
+    <div className="bg-black-106 border border-black-108 text-white-900 rounded-12 h-full"></div>
```

Comment:
Since this css is getting repeated across the focusroom page, consolidate it in a single place.
I mean, in index.css add this 
```
.focus-room-widget-border {
  @apply bg-black-106 border border-black-108 text-white-900 rounded-12 h-full;
}
```
and use `focus-room-widget-border` in all the widget where this series of class is repeated.